### PR TITLE
Update output.tf

### PR DESCRIPTION
**Pull request by enricomitri**
Added useful output definitions to retrieve the server which has the database installed

### CODE REVIEW

File: modules/databases/mssql_database/output.tf [hunk 1]
```diff
@@ -4,4 +4,16 @@ output "id" {
...
-}
\ No newline at end of file
+}
+
+output "server_id" {
+  value = azurerm_mssql_database.mssqldb.server_id
+}
+
+output "server_name" {
+  value = "${var.server_name}"
```
Comment (arnaudlh) [inline modules/databases/mssql_database/output.tf:14]: Feels a bit 2018 isnt it? :)

**Review by arnaudlh — CHANGES_REQUESTED** [review]

File: modules/databases/mssql_database/output.tf (hunk 1 above)
Comment (enricomitri) [reply modules/databases/mssql_database/output.tf:14]: Hi Arnaud,
thanks for your reply. I don't understand what you're asking, do you have an alternative solution?
Sorry this is my first terraform modules experience :)

Comment (arnaudlh) [reply modules/databases/mssql_database/output.tf:14]: hi @enricomitri, no problem we are all here learning! usually ```"${var.server_name}"``` is the Terraform former variables interpolation, nowadays we would simply ```var.server_name```

Comment (enricomitri) [reply modules/databases/mssql_database/output.tf]: Hi @arnaudlh, sure thanks for your explanation! I changed the definition as suggested. Let me know if it's ok

**Review by LaurentLesle — APPROVED** [review]
Thanks @enricomitri  for your first contribution

**Review by arnaudlh — APPROVED** [review]

**Pull request merged**